spark/dl/pom.xml

```diff
@@ -142,6 +142,7 @@
                     <systemProperties>
                         <bigdl.disableCheckSysEnv>true</bigdl.disableCheckSysEnv>
                     </systemProperties>
+                    <argLine>-Xmx6g</argLine>
```

Comment:
This is high, but I couldn't get all the tests to finish without running out of memory without this, like `GoogleNet_v2 model in batch mode`. 5GB might barely work.